<launch>
  <arg name="can_port" default="vcan0"/>
  <arg name="dbc_path" default="/usr/local/share/dbc/excavator_com3.dbc"/>
  
  <node pkg="excavator_com3_ros" name="lever_cmd_relay" type="lever_cmd_relay" output="screen">
    <param name="can_port" type="string" value="$(arg can_port)"/>
    <param name="dbc_path" type="string" value="$(arg dbc_path)"/>      
  </node>
  
  <node pkg="excavator_com3_ros" name="machine_setting_cmd_relay" type="machine_setting_cmd_relay" output="screen">
    <param name="can_port" type="string" value="$(arg can_port)"/>
    <param name="dbc_path" type="string" value="$(arg dbc_path)"/>      
  </node>

</launch>